typescript/src/lib/constructor_calls.ts

```diff
@@ -174,23 +174,41 @@ export function newPartitionConnections(allConnectionInfo: Map<string, Partition
     return result;
 }
 
-export function newWaitForEndpointAvailabilityArgs(
+export function newWaitForEndpointAvailabilityHttpGetArgs(
```

Comment:
Nice job adding these constructors!! Make sure to just propagate the rename swap here